libdokan/rekey_file.go

```diff
@@ -24,7 +24,9 @@ func (f *RekeyFile) WriteFile(ctx context.Context, fi *dokan.FileInfo, bs []byte
 	if len(bs) == 0 {
 		return 0, nil
 	}
-	err = f.folder.fs.config.KBFSOps().Rekey(ctx, f.folder.getFolderBranch().Tlf)
+	// TODO: get rid of this and use RequestRekey instead
```

Comment:
> this needs to be synchronous

Did you mean just the Rekey file or somewhere else as well?

Also, I suppose we are adding back `KBFSOps().Rekey()`, which is a "synchronous version" that calls "RequestRekey" and wait for first finish event?